CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 - Added distributed tracing using open telemetry APIs for table stored procedure function in `DataFrame`:
   - _execute_and_get_query_id
+- Avoid casting for numeric sql values for binary expression, multiExpression and InExpression. The change is controlled by session parameter _eliminate_numeric_sql_value_cast_enabled.
```

Comment:
We can mention performance improvement for binary column operations and dataframe._in method (users don't know about internal expressions). Also, we should mention the public parameter, like "this performance optimization can be enabled by setting `session.eliminate_numeric_sql_value_cast_enabled=True`"

Reply:
It seems the first part is not addressed

Reply:
sorry, updated to refect the user facing expressions